src/main/java/uk/gov/hmcts/probate/transformer/CaveatCallbackResponseTransformer.java

```diff
@@ -123,12 +123,18 @@ private ResponseCaveatDataBuilder getResponseCaveatData(CaveatDetails caveatDeta
                 .registryLocation(ofNullable(caveatData.getRegistryLocation()).orElse(DEFAULT_REGISTRY_LOCATION))
                 .deceasedForenames(caveatData.getDeceasedForenames())
                 .deceasedSurname(caveatData.getDeceasedSurname())
-                .deceasedDateOfDeath(dateTimeFormatter.format(caveatData.getDeceasedDateOfDeath()))
+                .deceasedDateOfDeath(caveatData.getDeceasedDateOfDeath() != null ? dateTimeFormatter.format(caveatData.getDeceasedDateOfDeath()) : null)
                 .deceasedDateOfBirth(transformToString(caveatData.getDeceasedDateOfBirth()))
                 .deceasedAnyOtherNames(caveatData.getDeceasedAnyOtherNames())
                 .deceasedFullAliasNameList(caveatData.getDeceasedFullAliasNameList())
                 .deceasedAddress(caveatData.getDeceasedAddress())
 
+                .solsSolicitorFirmName(caveatData.getSolsSolicitorFirmName())
```

Comment:
Have a chat with Janet about how we reuse the existing caveatorEmailAddress + caveatorAddress for the CW journey